kubeflow/apps/pipeline/Makefile

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 Google LLC
```

Comment:
Should we update to The Kubeflow Authors?

This repository might be different, because it is Google specific. But I want to confirm.

Reply:
Agree, stick to Google LLC